sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonExternalTransform.java

```diff
@@ -449,8 +449,7 @@ public OutputT expand(InputT input) {
       } else {
         int port = PythonService.findAvailablePort();
         ImmutableList.Builder<String> args = ImmutableList.builder();
-        args.add(
-            "--port=" + port, "--fully_qualified_name_glob=*", "--default_pickler=cloudpickle");
+        args.add("--port=" + port, "--fully_qualified_name_glob=*", "--pickle_library=cloudpickle");
```

Comment:
I'm also curious if this will break a bunch of tests.  Maybe in the portable version of beam it is fine.

Reply:
this only applies to external transforms where Python transform is called from Java. this is new functionality and there are not a lot of tests to break.